rotkehlchen/db/history_events.py

```diff
@@ -437,6 +438,7 @@ def get_history_events(
             filter_query: HistoryEventFilterQuery | EvmEventFilterQuery | EthDepositEventFilterQuery | EthWithdrawalFilterQuery,  # noqa: E501
             has_premium: bool,
             group_by_event_ids: bool = False,
+            msg_aggregator: 'MessagesAggregator | None' = None,
```

Comment:
hmm what are you doing here? Why are you passing the message aggregator here and in the rest? Please remove